SQF/dayz_code/Configs/CfgMagazines/Magazines/762x51.hpp

```diff
@@ -69,4 +69,28 @@ class 200Rnd_762x51_M240 : 100Rnd_762x51_M240 {
 	displayName = "200Rnd. M240";
 	count = 200;
 	descriptionShort = "Caliber: 7.62x51mm NATO <br/>Rounds: 200 <br/>Used in: M240, Mk 48 Mod 0";
-};
\ No newline at end of file
+};
+
+class 20Rnd_762x51_B_SCAR : CA_Magazine
+{
+	displayName = $STR_DZ_MAG_20Rnd_762x51_B_SCAR_NAME;
+	
+	cartridgeName = "762x51";
+	
+	class ItemActions
+	{
+		COMBINE_MAG
+	};
+};
+
+class 20Rnd_762x51_SB_SCAR : CA_Magazine
+{
+	displayName = $STR_DZ_MAG_20Rnd_762x51_SB_SCAR_NAME;
+	
+	cartridgeName = "762x51";
```

Comment:
Following the convention of the other configs, this should be "762x51_SB" to distinguish it from non-silenced I think.